unstable/wlr-foreign-toplevel-list.xml

```diff
@@ -0,0 +1,217 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<protocol name="wlr_foreign_toplevel_list_v1">
```

Comment:
The name should be `wlr_foreign_toplevel_list_unstable_v1`

Reply:
I guess I should rename the file, too?